mcs/build/profiles/basic.make

```diff
@@ -9,6 +9,8 @@ use_monolite := $(wildcard $(monolite_flag))
 
 MONOLITE_MSCORLIB = $(monolite_path)/mscorlib.dll
 
+PLATFORMS = macos linux win32 unix
```

Comment:
This is it what you want to do. Same issue as https://github.com/mono/mono/pull/8984#discussion_r192228194

Reply:
This review is not staled. This is not the way to go, fix it.